app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    Post.update(ComentsCounter: 1)
+  end
```

Comment:
### :red_circle: **[MANDATORY CHANGE]**

- [ ] It would be nice if you make your update_comments_counter as you can see below because the method you make is updating all posts' comments counter but it should only update the comments counter of that respective post for which any user has created a comment. So it would be great if you make your method as below :+1:

```ruby
  def update_comments_counter
    post.increment!(:ComentsCounter)
  end
```



### :yellow_circle: **[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new like has been saved to the database, then make the update_comments_counter method private method :+1: You can see below what I am talking :+1:

```ruby
after_save :update_comments_counter

  private

  def update_comments_counter
    post.increment!(:CommentsCounter)
  end
```